otaclient/app/proto/_common.py

```diff
@@ -232,11 +233,27 @@ def convert(
         return res
 
     def export_pb(self) -> Dict[_K, Any]:
+        warnings.warn(
```

Comment:
@Bodong-Yang 
Is this warn required?
I think abort is enough.

Reply:
Abort, do you mean directly return?

Reply:
No, this function is internal, and if this MessageMapContainer::export_bp is called, it is bug.
So raising exception or something should be done, if I'm undersdanding it correctly.